content/en/docs/reference/faq.md

```diff
@@ -0,0 +1,231 @@
+---
+title: "FAQ"
+linkTitle: "FAQ"
+weight: 5100
+description: >	
+    The most common problems and solutions.
```

Comment:
Description can be removed like this `description:`
And the FAQ should  be the last item in reference by adjusting weight